### ISSUE TITLE
Add CocoaPods spec

### ISSUE THREAD

**Issue by Kentzo** [Kentzo/ShortcutRecorder#7]

**Comment by Kentzo** [comment]
As of 0.36, [CocoaPods](http://blog.cocoapods.org/CocoaPods-0.36/) seems to support frameworks. Unfortunately documentation is incomplete and misleading, so I'm looking for a developer who is experienced enough with that utility.

**Comment by orta** [comment]
You should look at using `NSBundle`'s `bundleForClass` to figure out where to get resources from :+1:

**Comment by Kentzo** [comment]
@orta Does that mean I have to pack all resources in a bundle? Do you know of a cocoapods-compatible project I can use as a reference?

**Comment by orta** [comment]
There are tens of thousands of libraries on CocoaPods, here's a search for views that are OS X only, https://cocoapods.org/?q=on%3Aosx%20view - from a quick glance, MIHSliderView might be what you're looking for

**Comment by Kentzo** [comment]
@orta All resources of the ShortcutRecorder should be in its own bundle to avoid collisions. However, bundle created from the `resource_bundles` specification does not have any classes or even Info.plist (so no identifier).

How one is supposed to load anything from there?

**Comment by Kentzo** [comment]
Here is the podspec btw:

```
Pod::Spec.new do |s|
  s.name         = 'ShortcutRecorder'
  s.homepage = "https://github.com/Kentzo/ShortcutRecorder"
  s.summary = ""
  s.version      = '2.13'
  s.source       = { :git => 'git://github.com/Kentzo/ShortcutRecorder.git',
                     :branch => 'master' }
  s.author       = { 'Ilya Kulakov' => 'kulakov.ilya@gmail.com' }
  s.source_files = 'Library/*.{h,m}'
  s.frameworks   = 'Carbon', 'Cocoa'
  s.resource_bundles    = { "ShortcutRecorder" => ['Resources/*.lproj', 'Resources/*.png'] }
  s.requires_arc = true
  s.prefix_header_file = 'Library/Prefix.pch'
  s.platform     = :osx, "10.6"

  s.subspec 'PTHotKey' do |hotkey|
    hotkey.source_files = 'PTHotKey/*.{h,m}'
    hotkey.requires_arc = false
  end
end
```

**Comment by Kentzo** [comment]
Added basic podspec. Please test and let me know if there are any issues.

**Comment by gcox** [comment]
This podspec works fine for me :+1: 

I'm using both ShortcutRecorder and the subspec for PTHotKey.  The only thing I'd like to see is an umbrella header for PTHotkey.

**Comment by Kentzo** [comment]
@gcox Could you make a PR or point me to the related docs?

**Comment by gcox** [comment]
@Kentzo There's no docs really, it's just a header file that includes an ```#import``` statement that includes all the public headers...you've already got one in ShortcutRecorder.h, though I haven't checked to see if it actually includes **all** public headers.

I can make a PR but I will need to ask what to name the header file since ```PTHotKey.h``` is already taken and would be the usual name to go wth.

I wouldn't postpone submitting this to the Cocoa pods trunk because of my minor request...it is for convenience only.

**Comment by Kentzo** [comment]
Oh, I thought you were talking about a parameter in podspec. Sure, I can add this one.

**Comment by Kentzo** [comment]
Merged in 2.13
